contracts/standard/permission/ArbitrablePermissionList.sol

```diff
@@ -51,14 +50,14 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
      *  @param challenger Address of the challenger, if any.
      *  @param value The value of the item.
      *  @param newStatus The new status of the item.
-     *  @param newDisputed The new disputed state of the item.
+     *  @param disputed The item is being disputed.
```

Comment:
Doesn't seem like there is any logic around it being newly disputed. It always just broadcasts whether the item is currently disputed.